packages/ketcher-react/src/script/ui/component/form/input.jsx

```diff
@@ -25,8 +25,14 @@ export function GenericInput({
   value = '',
   onChange,
   type = 'text',
+  isFocus = false,
```

Comment:
Maybe the name should be "isFocused"? since its describing a particular state.

Reply:
Thank you, changed